task/upload.go

```diff
@@ -228,8 +230,10 @@ func processCatalystCallback(tctx *TaskContext, callback *clients.CatalystCallba
 			return nil, fmt.Errorf("output type mismatch: %s != %s", output.Type, outReq.Type)
 		}
 		manifestPath, err := extractOSUriFilePath(output.Manifest, playbackID)
-		if err != nil {
-			return nil, fmt.Errorf("error extracting file path from output manifest: %w", err)
+		if outName != OutputNameAssetMP4 {
```

Comment:
What is this check for? I don't think we have any output named that anymore, do we? Should probably remove the const and any references to it in the code.